parser.go

```diff
@@ -1029,9 +1029,10 @@ func collectIntegerValue(valueType *tokenType, value *any, until *rune) {
 
 func collectString() string {
 	var collection strings.Builder
+	escaped := false
```

Comment:
Please use this syntax to declare variables per the code style in the contribution guidelines. https://github.com/electrikmilk/cherri/blob/main/CONTRIBUTING.md

```suggestion
	var escaped = false
```